_includes/parse-server/file-adapters.md

```diff
@@ -1,3 +1,31 @@
+# Configuring File Upload
+
+*Available only on Parse Server starting 5.0.0*
+
+Parse Server restricts file upload to authenticated users only to improve Parse Server's default security. This behaviour can be modified by specifying `fileUpload` options to your Parse Server configuration.
+
+Available options are:
+
+`enableForAnonymousUser`: Whether file upload should be allowed for anonymous users
+`enableForAuthenticatedUser`: Whether file upload should be allowed for authenticated users
+`enableForPublic`: Whether file upload should be allowed for public users
```

Comment:
```suggestion
- `enableForAnonymousUser`: Whether file upload should be allowed for anonymous users
- `enableForAuthenticatedUser`: Whether file upload should be allowed for authenticated users
- `enableForPublic`: Whether file upload should be allowed for public users
```

Reply:
Did we really call it `enableForPublic` instead of `enableForPublicUser`?

Reply:
That is what we named it!